reservation/views.py

```diff
@@ -65,21 +69,62 @@ def select_seats(request, movie, schedule):
                 details=schedule_object,
                 seat=seat_obj,
             )
+
             reservations.append(user_reservation)
 
-        send_tickets(request, reservations, schedule_object)
-        return render(request, 'reservation/success.html')
+        confirm_reservation(request, reservations, schedule)
+        return render(request, 'reservation/reservation_email_sent.html')
 
     return render(request, 'reservation/select_seats.html', context)
 
 
-def send_tickets(request, reservation_ids, schedule):
+def success(request, group):
+
+    reservations = list(map(int, group.split('-')))
+
+    for reservation in reservations:
+        Reservation.objects.filter(
+            id=reservation
+        ).update(confirmed=True)
+
+    send_tickets(request, reservations)
+
+    return render(request, 'reservation/success.html')
+
+
+def send_tickets(request, reservations):
     subject, from_email, to = 'Your tickets', 'office@silverscreen.com', (request.user.email,)
     email_text = get_template('reservation/reservation_successful.html')
+
+    tickets = [Reservation.objects.get(id=reservation) for reservation in reservations]
```

Comment:
The same thing to note here, regarding the lookup.
Also, you're already retrieving the reservation objects based on their ids in the success function. Why not save them in a variable, call update on the variable, and send the variable (query set) directly to the send_tickets function?